runtime/v2/go.mod

```diff
@@ -0,0 +1,185 @@
+module cosmossdk.io/runtime/v2
+
+go 1.22.2
+
+// server v2 integration
+replace (
+	cosmossdk.io/api => ../../api
+	cosmossdk.io/core => ../../core
+	cosmossdk.io/depinject => ../../depinject
+	cosmossdk.io/server/v2/appmanager => ../../server/v2/appmanager
+	cosmossdk.io/server/v2/stf => ../../server/v2/stf
+	cosmossdk.io/store/v2 => ../../store/v2
+	cosmossdk.io/x/accounts => ../../x/accounts
+	cosmossdk.io/x/auth => ../../x/auth
+	cosmossdk.io/x/bank => ../../x/bank
+	cosmossdk.io/x/consensus => ../../x/consensus
+	cosmossdk.io/x/distribution => ../../x/distribution
+	cosmossdk.io/x/staking => ../../x/staking
+	cosmossdk.io/x/tx => ../../x/tx
```

Comment:
This coupling feels like a bad smell as well.

Reply:
these module imports are 2nd order (indirect) from importing `github.com/cosmos/cosmos-sdk` which is the root code.  I think the only way to avoid this is the creation of a `codec/v2/go.mod` to decouple the codec package usages.